Automattic/newspack-theme
# Query Block: Update styles to match theme, pick up custom fonts

### All Submissions:


### Changes proposed in this Pull Request:

WordPress 5.8 introduces a Query block, and a number of 'child' blocks, to let you build post queries and display post titles, images, taxonomies, etc. 

This PR updates the styles for these blocks to match the theme -- they're not 1:1 with the theme/homepage posts blocks (for example, the category and tags share the same CSS class, so they can't be styled the same as the theme out of the box, and the `|` used for dividers are hardcoded), but these updates should help them blend in more. 

See #1374

### How to test the changes in this Pull Request:

1. Start with a test site running the latest WordPress 5.8 RC (as of this ticket, its RC3). 
2. Add the Query block and pick a starting layout, or start fresh; you'll need to make sure the following blocks are included to test everything (note: you can also start with a Post List block; at the moment it seems the same as the Query block - best I can tell the plan is to turn it into a more user-friendly skin for the Query block, but at the moment they have the same options): 
     * Post Title
     * Post Content/Post Excerpt
     * Post Date
     * Post Featured Image
     * Post Categories
     * Post Tags
3. View on the front-end; the title and content/excerpt should pick up the fonts as expected, but elements like the categories, tags, dates, and pagination don't match how they're styled in the theme:

![image](https://user-images.githubusercontent.com/177561/125666005-ce017d53-d710-4de8-a2b3-2c7b57f77351.png)

![image](https://user-images.githubusercontent.com/177561/125665978-b6c4399c-8c5c-40d7-8471-72c54f265e19.png)

4. Apply the PR and run `npm run build`
5. Confirm that the different post elements are now picking up the theme's fonts and styles - post meta should use the header font and be a bit smaller than standard text; the pagination should match what's used on the archives and search:

![image](https://user-images.githubusercontent.com/177561/125665813-3178c71d-1cde-40c4-be3f-de44f15135f5.png)

![image](https://user-images.githubusercontent.com/177561/125665887-4fe459db-b6c5-4f03-bd05-11328681e2c0.png)

6. Make sure the same styles are being picked up in the editor, too. 

### Other information:
